data/brands/shop/convenience.json

```diff
@@ -3828,6 +3828,17 @@
         "shop": "convenience"
       }
     },
+    {
+      "displayName": "Tambo",
+      "id": "tambo-853840",
+      "locationSet": {"include": ["pe"]},
+      "tags": {
+        "brand": "Tambo",
+        "brand:wikidata": "Q116369947",
+        "name": "Tambo",
```

Comment:
Hi @whubsch , may I know the convenience's name is `Tambo` or `Tambo+` in Peru? Because there named `Tambo+` in both Wikidata and Wikipedia.

Reply:
Yeah good catch, it should be `Tambo+`, as long as that would still match with existing POIs already named just `Tambo`. Thanks.

Reply:
If so, you'de better add `Tambo` as a matchName, and I'll do this

Reply:
Thanks for your great help @LaoshuBaby